go/api/config.go

```diff
@@ -0,0 +1,290 @@
+// Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0
+
+package api
+
+import "github.com/aws/glide-for-redis/go/glide/protobuf"
+
+const (
+	defaultHost = "localhost"
+	defaultPort = 6379
+)
+
+// NodeAddress represents the host address and port of a node in the cluster.
+type NodeAddress struct {
+	Host string // Optional: if not supplied, "localhost" will be used.
```

Comment:
move the Optional comment above the field, rather than inline

Reply:
The [Go docs](https://tip.golang.org/doc/comment) have examples with both inline and above, but if we prefer them above then I can change